src/db/migrations/20230907143204_add_onerep_scanresults_table.js

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export async function up(knex) {
+  await knex.schema.alterTable("onerep_scans", table => {
+    table.unique("onerep_scan_id");
+  });
+
+  await knex.schema
+    .createTable("onerep_scan_results", table => {
+      table.increments("id").primary();
+      table.integer("onerep_scan_result_id").notNullable();
+      table.integer("onerep_scan_id").references("onerep_scans.onerep_scan_id").notNullable();
+      table.string("link").notNullable();
+      table.integer("age").nullable();
+      table.string("data_broker").notNullable();
+      table.integer("data_broker_id").notNullable();
+      table.jsonb("emails").notNullable();
+      table.jsonb("phones").notNullable();
+      table.jsonb("addresses").notNullable();
+      table.jsonb("relatives").notNullable();
+      table.string("first_name").notNullable();
+      table.string("middle_name").nullable();
+      table.string("last_name").notNullable();
+      table.string("status").notNullable();
+      table.timestamp("created_at").defaultTo(knex.fn.now());
+      table.timestamp("updated_at").defaultTo(knex.fn.now());
+      table.index("onerep_scan_id");
+      table.index("onerep_scan_result_id");
+    });
+
+  const scanRows = await knex("onerep_scans")
+    .select("id", "onerep_scan_id", "onerep_scan_results");
+
+  const scanResultRows = scanRows.map(scan => {
```

Comment:
Thanks for doing the migration here as well.
One weird issue we *might* encounter when we run this in heroku... which isn't your fault.. but unfortunately is that we might already have duplicate `onerep_scan_id` in the db.. which will stop this migration. We'll need to check heroku's db and probably manually clean up the data before pushing this to main

Reply:
Hmm, alternatively, I can just drop the latest versions of those? I'm assuming, especially since it's just Heroku, that losing the later versions isn't a big issue? Added in 10e700cd1, but happy to revert if you prefer going into the Heroku DB itself. (But if so, what would you prefer me to do with it?)